codebuild/cd/fips-aarch64-build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -ex
+
+cd $(dirname $0)/../..
+
+git submodule update --init
+# use hte FIPS approved branch.
+cd crt/aws-lc/ && git checkout fips-2022-11-02 && cd ../../
```

Comment:
I don't think we should have 2 submodules, I can have the crt-java just use the fips branch for non-fips build as well. We can just check to make sure we are using the fips branch probably

Reply:
Also, the FIPS branch for aws-lc is relatively safer, since they are not actively develop on the FIPS branch. They only backport the needed change to the FIPS branch from main.

Reply:
If the fips branch is fine for non-fips, then yeah let's just always use it.